docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -9,57 +9,58 @@ menu:
     parent: known-issues
     weight: 101
 type: docs
-rightNav:
```

Comment:
Can we have the right-side nav bar to be pinned as we scroll through the page? Right now, it does not seem to do that...

Reply:
There was some issue with right nav, it is fixed in production Once we merge the main changes, this pr should also reflect the sticky nav.